src/app.ts

```diff
@@ -0,0 +1,39 @@
+import express, { Application } from "express";
+import { Container } from "typedi";
+import { environment } from "./config/environment";
+import { Loaders } from "./loaders";
+import { Logger } from "./loaders/logger";
+
+class Server {
+  private app: Application = express();
+  private logger: Logger = Container.get(Logger);
+
+  constructor() {
+    this.init();
+  }
+
+  init() {
+    try {
+      new Loaders(this.app).init();
+      this.app.disable("x-powered-by");
+      this.app.listen(environment.port, err => {
+        if (err) {
+          this.logger.error(err);
+          process.exit(1);
+          return;
```

Comment:
Return is not reachable due to process.exit()
https://nodejs.org/api/process.html#process_process_exit_code

Reply:
Done.